server/plugin.ts

```diff
@@ -46,12 +46,19 @@ import { createMigrationOpenSearchClient } from '../../../src/core/server/saved_
 import { SecuritySavedObjectsClientWrapper } from './saved_objects/saved_objects_wrapper';
 import { addTenantParameterToResolvedShortLink } from './multitenancy/tenant_resolver';
 import { ReadonlyService } from './readonly/readonly_service';
+import { DataSourceManagementPlugin } from '../../../src/plugins/data_source_management/public/plugin';
+import { DataSourcePluginSetup } from '../../../src/plugins/data_source/server/types';
 
 export interface SecurityPluginRequestContext {
   logger: Logger;
   esClient: ILegacyClusterClient;
 }
 
+export interface SecurityPluginSetupDependencies {
+  dataSourceManagement: ReturnType<DataSourceManagementPlugin['setup']>;
```

Comment:
are we using this value somewhere?

Reply:
No I guess not, I removed it, may have been copied from somewhere